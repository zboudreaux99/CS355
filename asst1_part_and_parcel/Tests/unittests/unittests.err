Failed to map supported failure 'org.opentest4j.AssertionFailedError: expected: <null> but was: <[[abc, def], [ghi]]>' with mapper 'org.gradle.api.internal.tasks.testing.failure.mappers.OpenTestAssertionFailedMapper@38467116': Cannot invoke "Object.getClass()" because "expectedValue" is null

15 tests completed, 1 failed
