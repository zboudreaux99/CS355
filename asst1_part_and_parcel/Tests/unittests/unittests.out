> Task :compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava UP-TO-DATE
> Task :processTestResources NO-SOURCE
> Task :testClasses UP-TO-DATE

> Task :test

TestParser > testFile2() FAILED
    org.opentest4j.AssertionFailedError at TestParser.java:152
There were failing tests. See the report at: file:///home/cs_zboud002/CS355/asst1_part_and_parcel/build/reports/tests/test/index.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.5/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
3 actionable tasks: 1 executed, 2 up-to-date
